gef-ui/server/services/__tests__/api.test.js

```diff
@@ -388,3 +388,24 @@ describe('getAmendment()', () => {
     await expect(api.getAmendment({ facilityId: validMongoId, amendmentId: invalidMongoId, userToken })).rejects.toThrow('Invalid amendment ID');
   });
 });
+
+describe('upsertAmendment()', () => {
+  it(`should return ${HttpStatusCode.Ok} when a the amendment can be upserted`, async () => {
```

Comment:
Is this returning 200 just because thats what the mock value is - if thats the case should we be testing 'it returns the response body' or something like that?

Reply:
Such a good spot - have changed this (was a copy over legacy from another test doing similar things currently in this file..)